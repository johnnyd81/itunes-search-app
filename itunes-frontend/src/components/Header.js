import React from 'react'
import '../App.css'

const Header = () => {
  return (
    <div className="appHeader">
      <h1 className="titleText">iTunes search app</h1>
    </div>
  )
}

export default Header
