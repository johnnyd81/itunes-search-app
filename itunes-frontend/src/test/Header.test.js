import React from 'react'
import { create } from 'react-test-renderer'
// react test renderer has to be imported seperately as it is not included in the react testing library
import Header from '../components/Header'

describe('Header Component', () => {
  test('Matches the snapshot', () => {
    const header = create(<Header />)
    expect(header.toJSON()).toMatchSnapshot()
  })
})
