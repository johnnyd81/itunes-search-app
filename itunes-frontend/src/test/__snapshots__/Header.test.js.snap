// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component Matches the snapshot 1`] = `
<div
  className="appHeader"
>
  <h1
    className="titleText"
  >
    iTunes search app
  </h1>
</div>
`;
